CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-rc.9 September 22, 2021
+
+**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+
+## Features ✨ and improvements 🏁
+* Fix documentation for OnMapIdleListener and CameraChangeListeners. ([#645](https://github.com/mapbox/mapbox-maps-android/pull/645))
+* Add support for SymbolZOrder in PointAnnotationManager ([#638](https://github.com/mapbox/mapbox-maps-android/pull/638))
+* Add support for PromoteId to be used with Feature State API. ([#636](https://github.com/mapbox/mapbox-maps-android/pull/636))
+* Expose optimizeForTerrain flag ([#654](https://github.com/mapbox/mapbox-maps-android/pull/654))
+* Enable instant transitions for data driven symbol layer properties. ([#646](https://github.com/mapbox/mapbox-maps-android/pull/646))
+
+## Bug fixes 🐞
+* OnStyleLoaded / OnMapLoaded callbacks are invoked even if hosting fragment/activity is in stopped state. ([#629](https://github.com/mapbox/mapbox-maps-android/pull/629))
+* Fix drag annotation blink issue ([#639](https://github.com/mapbox/mapbox-maps-android/pull/639))
+* Apply annotation manager properties to drag layer to keep annotations the same while dragging ([#640](https://github.com/mapbox/mapbox-maps-android/pull/640))
```

Comment:
```suggestion
* Apply annotation manager properties to the drag layer to keep annotations the same while dragging. ([#640](https://github.com/mapbox/mapbox-maps-android/pull/640))
```

Reply:
@Chaoba was does `the same` means here? same comparing to what?

Reply:
Annotation properties like text color will change to default while dragging.